network-streaming-performance-k8s/README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Workflow Description
 
-This example workflow runs a network streaming bi-directional load using the
+This example workflow runs a network streaming one-directional workload using the
 [uperf](https://github.com/uperf/uperf) benchmark utility. The workload will
 run on a Kubernetes cluster using a provided `kubeconfig` object and will use a kubernetes service for communication between the uperf server and client pods.
 
-In addition to the uperf load, the workflow generates a UUID, which can be used as a unique key for the generated data, and runs a metadata collection using Ansible [gather facts](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/gather_facts_module.html).
+In addition to the uperf workload, the workflow generates a UUID, which can be used as a unique key for the generated data, collects system metrics with [Performance Co-pilot](https://pcp.io/), and collects system metadata using Ansible [gather facts](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/gather_facts_module.html).
```

Comment:
This system metrics and system metadata collection is a re-implementation of what other systems already do today.  I realize this is new, but we run the risk of proliferating re-implementations to work with Arcaflow if we are not careful.

Reply:
Truly the plugins that are in place today for metadata and metrics are rudimentary proof-of-concept implementations. Replacing them with plugins that better leverage existing implementations will likely be desired.

That being said, we should effectively be agnostic about the functions of individual plugins, and there isn't any reason that multiple approaches to what may look like the same challenge can't coexist.

The PR here is for a workflow to serve as a functional example, and so it is somewhat independent from the plugins it uses. I don't think that concerns about plugin implementations should necessarily block this workflow example PR.

Reply:
Yeah. We may package https://github.com/cloud-bulldozer/metadata-collector in a redhat-specific repository.